crates/networking/p2p/src/network.rs

```diff
@@ -406,9 +517,102 @@ mod tests {
         Network::init(executor, &config).await
     }
 
+    #[test]
+    fn peer_id_from_enr_matches_libp2p() {
+        let lp_kp = Keypair::generate_secp256k1();
```

Comment:
can you avoid shorthands in the tests you added as well, I won't point out every instance, but generally shorthands aren't readable for new contributors so we want to avoid them, to make everything as readable as possible